### PR TITLE
docs: Edits to remove jargon

### PR DESCRIPTION
**Pull request by TeriGlover**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [X] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: aio/content/guide/component-interaction.md [hunk 1]
```diff
@@ -296,13 +296,13 @@ Test also that clicking the *Stop* button pauses the countdown timer:
...
 
-The *local variable* approach is easy. But it is limited because
+The *local variable* approach is uncomplicated. But it is limited because
 the parent-child wiring must be done entirely within the parent template.
 The parent component *itself* has no access to the child.
 
 You can't use the *local variable* technique if the parent component's *class* relies on the
 child component's *class*.  The parent-child relationship of the components is not established
-within each components respective *class* with the *local variable* technique.  Since the *class*
+within each components respective *class* with the *local variable* technique.  Because the *class*
```
Comment (dario-piotrowicz) [inline aio/content/guide/component-interaction.md:305]: is there a reason why there are two spaces after the dot? :thinking: 
(yeah I know it was already there :sweat:)

File: aio/content/guide/content-projection.md [hunk 2]
```diff
@@ -150,7 +150,7 @@ This example uses an `ng-container` attribute to simulate projecting a component
 
 <header>Reminder!</header>
 
-The `ng-container` element is a logical construct that you can use to group other DOM elements; however, the `ng-container` itself is not rendered in the DOM tree.
+The `ng-container` element is a logical construct that used to group other DOM elements; however, the `ng-container` itself is not rendered in the DOM tree.
```
Comment (dario-piotrowicz) [inline aio/content/guide/content-projection.md:153]: The `ng-container` element is a logical construct that **used** to group other DOM elements

As in: `ng-container` used to do group elements but it doesn't anymore? :thinking: 

should it not be: 
The `ng-container` element is a logical construct that **is used** to group other DOM elements

or something of that sort?

File: aio/content/guide/inputs-outputs.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 # Sharing data between child and parent directives and components
 
 A common pattern in Angular is sharing data between a parent component and one or more child components.
-You can implement this pattern by using the `@Input()` and `@Output()` directives.
+Implement this pattern with the `@Input()` and `@Output()` directives.
```
Comment (AndrewKushnir) [inline aio/content/guide/inputs-outputs.md:4]: I'd propose replacing `directives` with `decorators` as it's technically more correct:
```suggestion
Implement this pattern with the `@Input()` and `@Output()` decorators.
```

**Review by AndrewKushnir — APPROVED** [review]
Looks great (just a couple minor comments) 👍

File: aio/content/guide/content-projection.md (hunk 2 above)
Comment (aikidave) [reply aio/content/guide/content-projection.md:153]: I think this should be "is used." Thanks for catching this, @dario-piotrowicz !

**Review by aikithoughts — APPROVED** [review]
LGTM, please address open comment before marking for merge.

Reviewed-for: global-docs-approvers

**Review by AndrewKushnir — APPROVED** [review]
👍

**Comment by josephperrott** [comment]
Caretaker Note:  This has been globally approved, but we are unable to remove @andrewseguin's review request.  This is ready to merge now.

**Pull request closed without merging**